robot_skills/test/test_arms.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python
+
+import sys
+import rospy
+
+import robot_skills.util.kdl_conversions as kdl_conversions
+
+if len(sys.argv) < 2:
+    print "Please specify a robot name"
+    sys.exit()
+
+robot_name = sys.argv[1]
+
+rospy.init_node("audio_test")
+if robot_name == "amigo":
+    from robot_skills.amigo import Amigo
```

Comment:
This code is duplicated a couple of times throughout tue_robocup. Maybe refactor/re-use?